### PR TITLE
added test-example order-handling.cy.ts

### PR DESCRIPTION
**Pull request by vadym-uskov**

### CODE REVIEW

File: cypress/support/scenarios/glue/checkout.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import { Carts } from 'cypress/support/glue-endpoints/cart/carts'
+import { Items } from 'cypress/support/glue-endpoints/cart/items'
+import { Checkout } from 'cypress/support/glue-endpoints/checkout/checkout'
+
+const tokenEndpoint = new AccessToken()
+const cartEndpoint = new Carts()
+const itemsEndpoint = new Items()
+const checkoutEndpoint = new Checkout()
+
+export class OrderDetailsPageScenario extends GlueRequest {
```
Comment (AntonShopin) [inline cypress/support/scenarios/glue/checkout.ts:12]: please rename it to `CheckoutScenario`

File: cypress/e2e/backoffice/order-handling.cy.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import userCredentials from '../../fixtures/user-credentials.json'
+import { BackofficeLoginPage } from '../../support/page-objects/backoffice/login/login-page'
+import { OrderListPage } from '../../support/page-objects/backoffice/order-management/order-list-page'
+import { OrderDetailsPage } from '../../support/page-objects/backoffice/order-management/order-details-page'
+import { OrderDetailsPageScenario } from 'cypress/support/scenarios/glue/checkout'
+
+const login = new BackofficeLoginPage()
+const orders = new OrderListPage()
+const order = new OrderDetailsPage()
+const orderDetailsPageScenario = new OrderDetailsPageScenario()
```
Comment (AntonShopin) [inline cypress/e2e/backoffice/order-handling.cy.ts:13]: please rename it to `checkoutScenario`

**Review by AntonShopin — COMMENTED** [review]
Once approved and merged -> please update the following wiki page as well: https://github.com/spryker-projects/cypress-boilerplate/wiki/Project-Structure

File: cypress/support/scenarios/glue/checkout.ts (hunk 1 above)
Comment (vadym-uskov) [reply cypress/support/scenarios/glue/checkout.ts]: fixed

File: cypress/e2e/backoffice/order-handling.cy.ts (hunk 2 above)
Comment (vadym-uskov) [reply cypress/e2e/backoffice/order-handling.cy.ts]: fixed

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**